src/Editor.tsx

```diff
@@ -35,11 +35,11 @@ export const Editor = ({doc, changeDoc}) => {
 
                                 changeDoc(
                                     (doc) => {
-                                        const layerToMove = doc.layers.splice(key, 1)[0]; 
+                                        const layerToMove = doc.layers.splice(key, 1)[0];
                                         doc.layers.splice(key-1, 0, layerToMove);
+                                        console.log(Array.from(doc.layers))
```

Comment:
Can you please get rid of this (and the other console.log below)?

Reply:
removed